docker/docker-compose.yml

```diff
@@ -32,8 +61,22 @@ services:
       retries: 5
     volumes:
       - postgres:/var/lib/postgresql/data
+
   redis:
-    image: redis:alpine
+    image: docker.io/library/redis:alpine
+    cap_drop:
+      - ALL
+    cap_add:
+      - DAC_OVERRIDE
+    security_opt:
+      - no-new-privileges:true
+    # uncomment to set resource usage limits
```

Comment:
Can you remove the resource usage limits? Anyone who needs it can configure it themselves.